templates/stripes/stripe-related.php

```diff
@@ -1,35 +1,50 @@
 <?php
-$links = array_values(array_filter($links, function ($link) {
-    return !empty($link['page_id']);
-}));
-?>
-<?php if (count($links)) : ?>
-<div class="<?= $this->e($type, 'wrapperClasses') ?>"<?php
-    echo !empty($variation) ? " data-variation=\"{$variation}\"" : '';
-?>>
+$links = array_values(
+  array_filter($links, function ($link) {
+    return !empty($link["page_id"]);
+  })
+); ?>
+<?php if (count($links)): ?>
+<div class="<?= $this->e($type, "wrapperClasses") ?>"<?php echo !empty(
+  $variation
+)
+  ? " data-variation=\"{$variation}\""
+  : ""; ?>>
     <div class="vssl-stripe-column">
-        <?php if (!empty($relatedLabel['html'])) : ?>
-        <h2 class="vssl-stripe--related--label"><?= $relatedLabel['html'] ?></h2>
-        <?php endif;?>
+        <?php if (!empty($relatedLabel["html"])): ?>
+        <h2 class="vssl-stripe--related--label"><?= $relatedLabel[
+          "html"
+        ] ?></h2>
+        <?php endif; ?>
         <div class="vssl-stripe--related--links">
-            <?php foreach ($links as $idx => $link) : ?>
+            <?php foreach ($links as $idx => $link): ?>
                 <div class="vssl-stripe--related--link">
-                    <a href="<?= $link['page']['slug'] ?>" target="_blank">
-                        <?php if (!empty($link['page']['image'])) : ?>
+                    <a href="<?= $link["page"]["slug"] ?>" target="_blank">
+                        <?php if (!empty($link["page"]["image"])): ?>
                         <img
                             class="vssl-stripe--related--thumbnail"
-                            src="<?= $this->image($link['page']['image'], !empty($image_style) ? $image_style : null) ?>"
-                            alt="<?= $link['page']['image'] ?>"
+                            src="<?= $this->image(
+                              $link["page"]["image"],
+                              !empty($image_style) ? $image_style : null
+                            ) ?>"
+                            alt="<?= $link["page"]["image_alt"] ??
```

Comment:
Using new $image_alt property on the referenced page for the alt attribute of the <img> tag, defaulting to the page's image's filepath if not set.

Reply:
My instinct would be to default to nothing if not set, since (a) no promises that the file name is readable, let alone meaningful, and (b) I'd categorize images on cards/links as often decorative, especially the text on the card should give you enough context to know what you're clicking on. What do you think?

Reply:
yup makes sense to me 👍